downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -121,6 +128,13 @@ enum class VcsHost(
 
         override fun toPermalinkInternal(vcsInfo: VcsInfo, startLine: Int, endLine: Int) =
             toGitPermalink(URI(vcsInfo.url), vcsInfo.revision, vcsInfo.path, startLine, endLine, "#L", "-L")
+
+        override fun getArchiveDownloadUrlInternal(vcsInfo: VcsInfo): String? {
+            val projectUrl = URI(vcsInfo.url.removeSuffix(".git"))
+            return projectUrlToUserOrOrgAndProject(projectUrl)?.let { (user, project) ->
+                "https://api.github.com/repos/$user/$project/tarball/${vcsInfo.revision}"
```

Comment:
Just FYI, the URL style I'm observing is like "https://codeload.github.com/oss-review-toolkit/ort/zip/advisor-vulners".

Reply:
No longer relevant